subtests/docker_cli/systemd/systemd_build.py

```diff
@@ -0,0 +1,23 @@
+"""
+Build docker image using Dockerfile and systemd unit file -
+``docker-build.service``. This unit file is copied to system and minor edit
+is made. A systemd service is started using this file to build the image.
+"""
+
+from systemd import systemd_base
+from dockertest.images import DockerImages
+
+
+class systemd_build(systemd_base):
+
+    def initialize(self):
+        super(systemd_build, self).initialize()
+        DockerImages(self).clean_all([self.config['image_name']])
```

Comment:
This should be assumed (clean environment).  It keeps the writers of ``cleanup()`` methods on their toes :grinning: